templates/travis/.travis/before_script.sh.j2

```diff
@@ -20,7 +20,10 @@ fi
 
 # Developers often want to know the final pulp config
 echo "PULP CONFIG:"
-$CMD_PREFIX bash -c "cat /etc/pulp/settings.*"
+$CMD_PREFIX bash -c "tail -v -n +1 /etc/pulp/settings.*"
+
+# Copy pulp config in a place accessible to the test code
+$CMD_PREFIX bash -c "cat /etc/pulp/settings.py" > $TRAVIS_BUILD_DIR/settings.py
```

Comment:
Glad to see this is helping you.

FYI: It is possible that people can change this (they can easily change the k8s cr.yaml that is used to generate settings.py) at runtime via POST_BEFORE_INSTALL, but this is an acceptable risk. We're about to use a single container anyway.